install.sh

```diff
@@ -29,26 +29,58 @@ function v() {
   fi
 }
 #####################################################################################
-printf "[$0]: Hi there!\n"
+startask (){
+printf "\e[34m[$0]: Hi there!\n"
 printf 'This script 1. only works for ArchLinux and Arch-based distros.\n'
 printf '            2. has not been fully tested, use at your own risk.\n'
-printf "\e[36m== PLEASE BACKUP \"$HOME/.config\" AND \"$HOME/.local\" BY YOURSELF IF NEEDED! ==\n\e[97m"
+printf '\n'
+printf "\e[31m"
+printf "=============\n"
+printf "PLEASE CONFIRM\n"
+printf "YOU HAVE\n"
+printf "  \"$HOME/.config\"\n"
+printf "  AND\n"
+printf "  \"$HOME/.local\"\n"
+printf "ALREADY BACKUPED\n"
```

Comment:
- "backed up" not "backuped"
- Also I don't think it's necessary to have this on so many lines. A non-white color that stands out should be enough.
- So I'd rather have something like this:
```
Please CONFIRM that you HAVE ALREADY BACKED UP $HOME/.config and $HOME/.local folders
Enter "YES" (in capitals, without quotes) to continue: 
```